tests/ClientTest.php

```diff
@@ -2,44 +2,61 @@
 
 namespace Solarium\Tests;
 
+use Composer\InstalledVersions;
 use PHPUnit\Framework\TestCase;
 use Solarium\Client;
 
 class ClientTest extends TestCase
 {
+    /**
+     * The version tag we use for testing within github actions.
+     *
+     * @var string
+     */
+    protected static $versionTag = '76.5.4';
+
+    public static function setUpBeforeClass(): void
+    {
+        $installedVersion = InstalledVersions::getPrettyVersion('solarium/solarium');
+
+        if ('dev-master' === $installedVersion) {
```

Comment:
We should use ```strpos($installedVersion, 'dev-') === 0``` to cover feature branches, too.

Reply:
That was temporary to test local vs. github for which I had to push. ;-)